content/posts/monorepos.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Introducing monorepository support
+date: 2023-07-04T09:20:06+02:00
+weight: 89
+---
+
+We are very happy to announce a major enhancement to Packit! We have now added support for
+monorepositories, enabling the integration of upstream repositories containing multiple downstream packages.
+If you have a repository in the monorepo format, Packit can now help you automate the integration to downstream
+distributions both from CLI and as a service.
+
+## Configuration
+
+Let's take a look at how a monorepository should be configured so that Packit can automate the process!
+
+The main addition to the Packit configuration file in the context of monorepositories are the keys
+[`packages`](/docs/configuration#packages) and [`paths`](/docs/configuration#paths).
+
+`packages` holds a dictionary of `{package_name: package_configuration}` where package configuration can contain any keys
+that were previously used as top-level keys for the standard (single) package configuration. The `paths` can be defined
+in each `package_configuration` and should hold a list of paths that should be considered for the particular package.
+
+The `packages` section in the configuration can then look like this:
+
+```yaml
+packages:
+  python-copr:
+    downstream_package_name: python-copr
+    upstream_package_name: copr
+    paths:
+      - ./python
+    specfile_path: python-copr.spec
+    files_to_sync:
+      - python-copr.spec
+
+  copr-cli:
+    downstream_package_name: copr-cli
+    upstream_package_name: copr-cli
+    paths:
+      - ./cli
+    specfile_path: copr-cli.spec
+    files_to_sync:
+      - copr-cli.spec
+
+  copr-backend:
+    downstream_package_name: copr-backend
+    upstream_package_name: copr-backend
+    paths:
+      - ./backend
+    specfile_path: copr-backend.spec
+    files_to_sync:
+      - copr-backend.spec
+```
+
+That was the configuration of the packages and their locations in general, but how should these values then be utilised?
+
+### Packit Service support
```

Comment:
What about adding a screenshot of the status to make it more eye-cathing (we can also use the same for Mastodon post that can market this post).

Reply:
Sadly, the failed status does not look so appealing but we don't need to spend so much time on that..;)

Reply:
Thanks